internal/cli/status.go

```diff
@@ -1098,7 +1098,7 @@ their current state, run ‘waypoint status -app=APP-NAME PROJECT-NAME’.
 `)
 
 	wpStatusAppSuccessMsg = strings.TrimSpace(`
-The application and its declared resources listed above represents its current state known
+The application and it's resources listed above represents it's current state known
```

Comment:
Totally incidental change -  we're currently displaying actual resources, not declared resources.

Reply:
FYI it still would be `its` -- the apostrophe in `it's` does not denote possession, only contraction.
```suggestion
The application and its resources listed above represents its current state known
```

Reply:
🤦‍♂️🙏

Reply:
Fixing in https://github.com/hashicorp/waypoint/pull/2062